cpp/monitoring/registry.cc

```diff
@@ -11,43 +15,20 @@ Registry* Registry::Instance() {
 
 
 void Registry::AddMetric(const Metric* metric) {
-  std::lock_guard<std::mutex> lock(mutex_);
+  lock_guard<mutex> lock(mutex_);
   metrics_.insert(metric);
 }
 
 
 void Registry::ResetForTestingOnly() {
-  std::lock_guard<std::mutex> lock(mutex_);
+  lock_guard<mutex> lock(mutex_);
   metrics_.clear();
 }
 
 
-void Registry::Export(std::ostream* os) const {
-  std::lock_guard<std::mutex> lock(mutex_);
-  for (const auto* m : metrics_) {
-    m->Export(os);
-  }
-}
-
-
-void Registry::ExportHTML(std::ostream* os) const {
```

Comment:
Previously the concrete `Metric` implementations knew how to render themselves to an `ostream` and this would just get all known metrics to do that, but now the exporter does that job.